pkg/agent/baker.go

```diff
@@ -26,17 +26,23 @@ func InitializeTemplateGenerator() *TemplateGenerator {
 	return t
 }
 
+// GetNodeBootstrappingPayloadNonARM get node bootstrapping data that's not to be used in an ARM template.
+func (t *TemplateGenerator) GetNodeBootstrappingPayloadNonARM(config *NodeBootstrappingConfiguration) string {
```

Comment:
I don't think we need to distinguish between the ARM and no ARM implementation, the VM's custom data string should always be base64 string, no matter how we create it

Reply:
Fixed now. Now we just have a single implementation that gets a base64 encoded bootstrapping custom data.